packages/@react-spectrum/tooltip/test/TooltipTrigger.test.js

```diff
@@ -469,7 +486,7 @@ describe('TooltipTrigger', function () {
       expect(tooltip).not.toBeInTheDocument();
     });
 
-    it('once opened, it can be closed and opened instantly for a period of time',  () => {
+    it.skip('once opened, it can be closed and opened instantly for a period of time',  () => {
```

Comment:
this test doesn't make a lot of sense now unless you'd like to specify different global and local cooldowns

Reply:
Maybe we change this test to use the `ESC` key to close it faster so we can check the cooldown

Reply:
good idea - I've just refactored this test using the advice 👍